lib/managers/files.js

```diff
@@ -257,16 +257,26 @@ Files.prototype.getReadStream = function(fileID, options, callback) {
  *
  * @param {string} fileID - Box ID of the file being requested
  * @param {Object} [options] - Additional options for the request. Can be left null in most cases.
+ * @param {string} [options.extension] - The file format for the thumbnail. Default is png.
  * @param {Function} [callback] - Passed the thumbnail file or the URL to a placeholder thumbnail if successful.
  * @returns {Promise<Object>} A promise resolving to the thumbnail information
  */
 Files.prototype.getThumbnail = function(fileID, options, callback) {
+	var extension = 'png';
+
+	if (options && options.extension) {		
+		if (['png', 'jpg'].indexOf(options.extension) !== -1) {
```

Comment:
Let's only set a new value for `extension` if `options.extension` **equals** 'jpg' exactly.  The current code could allow the extension to be set to an invalid extension, such as `bad-jpg-value`.  If a bad value like `bad-jpg-value` is passed in, then we should use `png` as the default.

Also, I see no reason to add `png` to the `options.extension` check, since it's set as the default on line 265.